coding/traffic.hpp

```diff
@@ -25,11 +25,16 @@ class TrafficGPSEncoder
   {
     DataPoint() = default;
 
-    DataPoint(uint64_t timestamp, ms::LatLon latLon) : m_timestamp(timestamp), m_latLon(latLon) {}
+    DataPoint(uint64_t timestamp, ms::LatLon latLon, uint8_t traffic)
+      : m_timestamp(timestamp), m_latLon(latLon), m_traffic(traffic)
+    {
+    }
     // Uint64 should be enough for all our use cases.
     // It is expected that |m_timestamp| stores time since epoch in seconds.
     uint64_t m_timestamp = 0;
     ms::LatLon m_latLon = ms::LatLon::Zero();
+    // TODO: comment the decision
```

Comment:
A pod type instead of the traffic::SpeedGroup enum so as not to introduce a cyclic dependency.
This filed was added in Version 1 (and was the only addition).